### ISSUE TITLE
Add link to API doc in GitHub readme.md

### ISSUE THREAD

**Issue by msawczyn** [VsixCommunity/Community.VisualStudio.Toolkit#220]
To spur adoption, add a DocFX site to github pages and link it prominently from the main project's readme.md

**Comment by madskristensen** [comment]
Yes, I would love to add docfx support and host it on github pages. In fact, this project used to do just that. But I couldn't get docfx to work properly, so I abandoned the idea. If anyone wants to give it a shot, that would be awesome!!!
